### PR TITLE
Explain contents of bundled JS files in docs (#23735)

### PR DESCRIPTION
**Pull request by vannitotaro**
See https://github.com/twbs/bootstrap/issues/23381#issuecomment-340402261 and https://github.com/twbs/bootstrap/issues/23381#issuecomment-340403219.

### CODE REVIEW

**Comment by vannitotaro** [comment]
@Johann-S here it is the PR, as per your request. :wink:

**Review by XhmikosR — DISMISSED** [review]
Please add the trailing slashes in the URLs

File: docs/4.0/getting-started/contents.md [hunk 1]
```diff
@@ -82,6 +82,36 @@ This is the most basic form of Bootstrap: precompiled files for quick drop-in us
   </tbody>
 </table>
 
+### Comparison of JS files
```
Comment (Johann-S) [inline docs/4.0/getting-started/contents.md]: Not sure if this table, isn't too much here for just two files

Comment (XhmikosR) [reply docs/4.0/getting-started/contents.md]: I believe it's just to match the style we have for the CSS above.

Comment (vannitotaro) [reply docs/4.0/getting-started/contents.md]: Yeah, maybe overkill but it matches the other table style. And I think it's easier to be modified, should other dependencies be added in the future.

**Comment by vannitotaro** [comment]
@XhmikosR I've added trailing slashes in the URLs as per your request. Thank you. :smiley:

**Review by mdo — APPROVED** [review]

**Comment by XhmikosR** [comment]
@mdo: you should have squashed this.

**Pull request merged**